packages/models-library/src/models_library/utils/nodes.py

```diff
@@ -65,6 +65,8 @@ async def compute_node_hash(
                 resolved_payload[port_type][port_key] = payload
 
     # now create the hash
+    # WARNING: Here we cannot change to json_serialization.json_dumps because if would create a different dump string and therefore a different hash
+    # NOTE that these hashes might have been already stored elsewhere
```

Comment:
Question here: this is a byte string, this should still work I think. if the string is the same then the hash will remain the same as well. Ok, you are right. I guess this would need some migration in order to work correctly.